x-pack/plugins/apm/public/components/app/correlations/ml_failure_correlations.tsx

```diff
@@ -0,0 +1,409 @@
+                  },
+                })}
+              >
+                <EuiIcon type="magnifyWithPlus" />
+              </EuiLink>
+              &nbsp;/&nbsp;
+              <EuiLink
+                href={createHref(history, {
+                  query: {
+                    kuery: `not ${term.fieldName}:"${encodeURIComponent(
+                      term.fieldValue
+                    )}"`,
+                  },
+                })}
+              >
+                <EuiIcon type="magnifyWithMinus" />
+              </EuiLink>
+            </>
+          );
+        },
+      },
+    ],
+    [handleOnFilter, history, trackApmEvent]
+  );
+
+  useEffect(() => {
+    if (isErrorMessage(error)) {
+      notifications.toasts.addDanger({
+        title: i18n.translate('xpack.apm.correlations.failure.errorTitle', {
+          defaultMessage:
+            'An error occurred performing correlations on failed transactions',
+        }),
+        text: error.toString(),
+      });
+    }
+  }, [error, notifications.toasts]);
+  return (
+    <>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          {!isRunning && (
+            <EuiButton size="s" onClick={startFetch}>
+              <FormattedMessage
+                id="xpack.apm.correlations.failure.refreshButtonTitle"
+                defaultMessage="Refresh"
+              />
+            </EuiButton>
+          )}
+          {isRunning && (
+            <EuiButton size="s" onClick={cancelFetch}>
+              <FormattedMessage
+                id="xpack.apm.correlations.failure.cancelButtonTitle"
+                defaultMessage="Cancel"
+              />
+            </EuiButton>
+          )}
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFlexGroup direction="column" gutterSize="none">
+            <EuiFlexItem data-test-subj="apmCorrelationsfailedTransactionsCorrelationsProgressTitle">
+              <EuiText size="xs" color="subdued">
+                <FormattedMessage
+                  data-test-subj="apmCorrelationsfailedTransactionsCorrelationsProgressTitle"
+                  id="xpack.apm.correlations.failure.progressTitle"
+                  defaultMessage="Progress: {progress}%"
+                  values={{ progress: Math.round(progress * 100) }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiProgress
+                aria-label={i18n.translate(
+                  'xpack.apm.correlations.failure.progressAriaLabel',
+                  { defaultMessage: 'Progress' }
+                )}
+                value={Math.round(progress * 100)}
+                max={100}
+                size="m"
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <LatencyCorrelationsHelpPopover />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      {selectedTerm ? (
+        <>
+          <Summary
```

Comment:
I wonder if there needs to be a spacer in between the progress bar row and the `EuiBadge` row displaying the p-value. I was expecting a tooltip when hovering over the score indicator, and took me a while to spot it here under the progress bar.

![image](https://user-images.githubusercontent.com/7405507/129387423-9a281e57-3ead-45d5-862e-2dbcb309a6c6.png)

Reply:
Added spacer here https://github.com/elastic/kibana/pull/108441/commits/cb6123208047c376c015f6fc733bbefe2d39035f